tests/static_data/__init__.py

```diff
@@ -555,7 +555,7 @@
             ]},
         ],
         TEST_DATA: {
-            'bbox': (-20026376.39, -20048966.1, 20026376.39, 18440002.895114236),
+            'bbox': (-20037508.342789244, -30240971.95838617, 20037508.342789248, 18440002.8951142476),
             'file_type': settings.FILE_TYPE_VECTOR,
             'style_type': 'sld',
             'micka_xml': {'filled_template': 'test_tools/data/micka/rest_test_filled_template.xml',
```

Comment:
File test_tools/data/thumbnail/small_layer_4326.png is not used

Reply:
It is used in tests/static_data/__init__.py in post_tif_rgba_4326